acpi/src/madt.rs

```diff
@@ -551,12 +551,15 @@ pub struct GiccEntry {
     performance_interrupt_gsiv: u32,
     parked_address: u64,
     gic_registers_address: u64,
-    gic_control_block_address: u64,
+    gic_virtual_registers_address: u64,
+    gic_hypervisor_registers_address: u64,
     vgic_maintenance_interrupt: u32,
     gicr_base_address: u64,
     mpidr: u64,
     processor_power_efficiency_class: u8,
-    _reserved2: [u8; 3],
+    _reserved2: u8,
+    spe_overflow_interrupt: u16,
```

Comment:
Spec doesn't have a comment for when this field was added - do you happen to know without having to look through every version of the spec?

Reply:
It was added in 6.3. However, the 'Revision' field of the MADT was not updated (I'm guessing because the GICC structure did not change in size, as the 'SPE overflow Interrupt' field just took the place of some reserved bytes), so I am unsure about how to use `ExtendedField`. The field is zeroed though if it is unsupported by the ACPI version or SPE is not supported by the processor, so maybe it is viable to plainly define it as a `u16`?

Reply:
Hm yeah if it is defined to be zeroed in prior versions that should be fine - documenting the field with the version at which it becomes valid would probably be enough.

Reply:
I added a bit of documentation to the field explaining the situation. It might not be of much use, though, because the fields are all private. Should they be exposed (by another pull request)?

Reply:
It's still useful to have these documented for future internal contributors.

But yeah, the option to just expose all the fields on these "raw" structs is one that has come up in a Discussion recently (#158) - I don't think I have strong preferences so if people think it would be useful to be able to mangle these themselves I'm happy with that.